buildSrc/src/main/java/dependencies/Dependencies.kt

```diff
@@ -0,0 +1,99 @@
+    const val burgstaller_okhttp_digest = "com.burgstaller:okhttp-digest:2.5"
+    const val persian_joda_time = "com.github.mohamadian:PersianJodaTime:1.2"
+    const val myanmar_calendar = "com.github.chanmratekoko:myanmar-calendar:1.0.6.RC3"
+    const val bikram_sambat = "bikramsambat:bikram-sambat:1.1.0"
+    const val danlew_android_joda = "net.danlew:android.joda:2.10.9.1"
+    const val rarepebble_colorpicker = "com.rarepebble:colorpicker:3.0.1"
+    const val commons_io = "commons-io:commons-io:2.5" // Commons 2.6+ introduce java.nio usage that we can't access until our minSdkVersion >= 26 (https://developer.android.com/reference/java/io/File#toPath())
+    const val opencsv = "net.sf.opencsv:opencsv:2.4"
+    const val javarosa = "org.getodk:javarosa:3.2.0"
+    const val karumi_dexter = "com.karumi:dexter:6.2.3"
+    const val zxing_android_embedded = "com.journeyapps:zxing-android-embedded:3.6.0" // Upgrading will require minSdkVersion >=24, it uses zxing:core 3.3.2 by default
+    const val dagger = "com.google.dagger:dagger:${Versions.dagger}"
+    const val dagger_compiler = "com.google.dagger:dagger-compiler:${Versions.dagger}"
+    const val dagger_android = "com.google.dagger:dagger-android:${Versions.dagger}"
+    const val dagger_android_support = "com.google.dagger:dagger-android-support:${Versions.dagger}"
+    const val dagger_android_processor = "com.google.dagger:dagger-android-processor:${Versions.dagger}"
+    const val kotlinx_coroutines_android = "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.5.1"
+    const val rxandroid = "io.reactivex.rxjava2:rxandroid:2.1.1"
+    const val rxjava = "io.reactivex.rxjava2:rxjava:2.2.21"
+    const val rxrelay = "com.jakewharton.rxrelay2:rxrelay:2.1.1"
+    const val armen101_audio_recorder_view = "com.github.Armen101:AudioRecordView:1.0.5"
+    const val glide = "com.github.bumptech.glide:glide:4.12.0"
+    const val mp4parser_muxer = "org.mp4parser:muxer:1.9.41"
+    const val butterknife = "com.jakewharton:butterknife:10.2.3"
+    const val butterknife_compiler = "com.jakewharton:butterknife-compiler:10.2.3"
+    const val kotlin_stdlib = "org.jetbrains.kotlin:kotlin-stdlib:${Versions.kotlin}"
+    const val kotlin_stdlib_jdk7 = "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${Versions.kotlin}"
+    const val json = "org.json:json:20210307"
+    const val gson = "com.google.code.gson:gson:2.8.7"
+    const val firebase_analytics = "com.google.firebase:firebase-analytics:19.0.0"
+    const val firebase_crashlytics = "com.google.firebase:firebase-crashlytics:18.2.1"
+    const val findbugs_annotations = "com.google.code.findbugs:annotations:3.0.1"
+    const val findbugs_jsr305 = "com.google.code.findbugs:jsr305:3.0.2"
+    const val fastlane_screengrab = "tools.fastlane:screengrab:2.1.0"
+    const val leakcanary = "com.squareup.leakcanary:leakcanary-android:2.7"
+    const val timber = "com.jakewharton.timber:timber:${Versions.timber}"
```

Comment:
I don't think Timber needs its own `Version` constant as it's only used once? Also, Timber was just updated to fix a bug we ran into (https://github.com/JakeWharton/timber/issues/434#event-5155819740). Lets update here and remove the ignore in `lint.xml` on `TimberArgCount`.

Reply:
Done.